README.md

```diff
@@ -1,12 +1,12 @@
 # Movies list - Filter
 - Replace `<your_account>` with your Github username in the
- [DEMO LINK](https://<your_account>.github.io/react_movies-list-filter/)
+ [DEMO LINK](https://WojciechCzarnowski.github.io/react_movies-list-filter/)
 - Follow the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline)
 - Use [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript)
 
 ## Task
 You are given movies list and a search field. Implement filtering using a `useState` hook.
-1. On every change save the input value into the `query` 
+1. On every change save the input value into the `query`
 2. Create a `visibleMovies` variable in with the filtered movies
```

Comment:
Functionality of your app is good, but you need to create `visibleMovies` array.